extensions/Bookalope/js/bookalope.js

```diff
@@ -579,10 +584,12 @@ function askSaveBookflowFile(bookflow, format, style, version) {
         .then(function (book) {
             var bookflow = book.bookflows[0];
             uploadFile(bookflow);
+            console.log('then')
```

Comment:
@nGolubeva, did you intend to push the logging? 😉

Reply:
=) Removed.